<!--REQUEST_ID=ba13e569-5965-46cd-84be-52b66261515e-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48873&attempt_id=60da7831-db69-4f4b-8a77-dfef1f2f106a)

Attempted: jwm-settings-manager

<details><summary>Partial log (click to expand)</summary><p>

```
                        ^
/nix/store/vdbia3wglg1y6pjshgsy1viz8r0k09bg-xproto-7.0.31/include/X11/X.h:96:13: note: previous definition is here
typedef XID Window;
            ^
1 error generated.
make[2]: *** [CMakeFiles/JSM.dir/build.make:63: CMakeFiles/JSM.dir/src/ui.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:110: CMakeFiles/JSM.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
builder for '/nix/store/bvyqdh89161n25hyjk0zgs43nxc8b86i-jwm-settings-manager-2018-10-19.drv' failed with exit code 2
error: build of '/nix/store/bvyqdh89161n25hyjk0zgs43nxc8b86i-jwm-settings-manager-2018-10-19.drv' failed
```
</p></details>

--- a/pkgs/applications/window-managers/jwm/jwm-settings-manager.nix
+++ b/pkgs/applications/window-managers/jwm/jwm-settings-manager.nix
@@ -1,0 +1,44 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, gettext, libXpm, libGL, fltk, hicolor-icon-theme, glib, gnome2, which }:
+
+stdenv.mkDerivation rec {
+  name = "jwm-settings-manager-${version}";
+  version = "2018-10-19";
+  
+  src = fetchFromGitHub {
+    owner = "Israel-D";
+    repo = "jwm-settings-manager";
+    rev = "cb32a70563cf1f3927339093481542b85ec3c8c8";
+    sha256 = "0d5bqf74p8zg8azns44g46q973blhmp715k8kcd73x88g7sfir8s";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkgconfig
+    gettext
+  ];
+
+  buildInputs = [
+    libXpm
+    libGL
+    fltk
+    hicolor-icon-theme
+    which # needed at runtime to locate optional programs
+    glib.bin # provides gsettings
+    gnome2.GConf # provides gconftool-2
+  ];
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace 'CMAKE_INSTALL_PREFIX "/usr"' "CMAKE_INSTALL_PREFIX $out"
+    substituteInPlace data/CMakeLists.txt \
+      --replace 'DESTINATION usr/share' "DESTINATION share"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A full configuration manager for JWM";
+    homepage = https://joewing.net/projects/jwm;
+    license = licenses.gpl3;
+    platforms = platforms.linux;
+    maintainers = [ maintainers.romildo ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17403,6 +17403,8 @@ with pkgs;
 
   jwm = callPackage ../applications/window-managers/jwm { };
 
+  jwm-settings-manager = callPackage ../applications/window-managers/jwm/jwm-settings-manager.nix { };
+
   k3d = callPackage ../applications/graphics/k3d {
     inherit (pkgs.gnome2) gtkglext;
     stdenv = overrideCC stdenv gcc6;

